packages/components/src/components/SquiggleChart.tsx

```diff
@@ -51,6 +51,8 @@ export interface SquiggleChartProps {
   minX?: number;
   /** Specify the upper bound of the x scale */
   maxX?: number;
+  /** Whether the x-axis should be dates or numbers */
+  xAxis?: "number" | "dateTime";
```

Comment:
This is quite a general parameter name, I think for now this should either be:

```typescript
xAxisType: "number" | "dateTime"
```
or

```typescript
xAxisDate: boolean
```

Reply:
Great point, changed over.